utils/runtime-fuzzer/src/gear_calls.rs

```diff
@@ -254,8 +256,8 @@ impl UploadProgramGenerator {
     }
 
     const fn unstructured_size_hint(&self) -> usize {
-        // Max code size - 50 KiB.
-        const MAX_CODE_SIZE: usize = 50 * 1024;
+        // Max code size - 25 KiB.
```

Comment:
Why have you changed this?

Reply:
As I got from logs, fuzzer doesn't create wasm (in current configuration) with size larger than 10 kb. So 25 kb is absolutely enough.